src/Categories/Category/Construction/Empty.agda

```diff
@@ -0,0 +1,13 @@
+{-# OPTIONS --without-K --safe #-}
+
+module Categories.Category.Construction.Empty {o ℓ e}  where
```

Comment:
I think this already exists as `Categories.Category.Instance.Zero` (although your implementation is a lot tidier!)

Reply:
I see! I wonder why everything was using `liftC (Discrete 0)` then.

any preference on a name to use for both of them?

Reply:
Please stick with the existing name

Reply:
ah, there is also `Categories.Category.Instance.EmptySet`
quite a few of these.

Reply:
`Categories.Category.Instance.EmptySet` is certainly in the wrong place, but it's not a duplicate: it defines the initial objects for the `Sets` and `Setoids` categories